server/routes/community.js

```diff
@@ -37,11 +39,9 @@ router.delete('/removeCommunityListing', (req, res) => {
 });
 
 // returns all community listings
-router.get('/getAllCommunityListings', (req, res) => {
-  return getAllCommunityListings()
-    .then(allCommunityListings => res.send(allCommunityListings))
-    .catch(err => console.log('server/community, error line 43: ', err));
-});
+router.get('/getAllCommunityListings', (req, res) => getAllCommunityListings()
```

Comment:
For consistency, I recommend writing functions that just return promises like this:
`router.get('/getAllCommunityListings', (req, res) => (
    getAllComunityListings()
    ....
);`